perception/radar_fusion_to_detected_object/schema/radar_fusion_to_detected_object.schema.json

```diff
@@ -0,0 +1,114 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Radar Fusion to Detected Object Node",
+  "type": "object",
+  "definitions": {
+    "radar_fusion_to_detected_object": {
+      "type": "object",
+      "properties": {
+        "node_params": {
+          "type": "object",
+          "properties": {
+            "update_rate_hz": {
+              "type": "number",
+              "description": "Update rate for parameters. [hz]",
+              "default": 10.0,
```

Comment:
Please use quotes for all `"default"` values, e.g.: `10.0` --> `"10.0"` or else it doesn't auto-populate the parameter file correctly. I missed this in my first review, sorry about that.

This applies to all parameters which have a `"default"` value.